src/features/customerView/__tests__/utils.test.ts

```diff
@@ -93,7 +93,7 @@ describe('utils', () => {
         berthLeases: null,
       };
 
-      const leases = getLeases(profile);
+      const leases = getBerthLeases(profile);
```

Comment:
The test `describe` block should be renamed `getBerthLeases`

Reply:
Done.

Reply:
Done.